DockerfileDocs

```diff
@@ -0,0 +1,9 @@
+FROM debian:8
+
+RUN apt-get update && \
+    apt-get install -y xsltproc libxml2-utils build-essential git curl python2.7
```

Comment:
to make the image a little bit slimmer, we could clean up after ourselves here, by adding `&& rm -rf /var/lib/apt/lists/*`